elementary/utils/log.py

```diff
@@ -25,7 +28,30 @@ def format(self, record):
 
 
 FORMATTER = ColoredFormatter()
-LOG_FILE = "edr.log"
+LOG_FILE = os.getcwd() + "/edr_files/edr.log"
+
+
+class CustomerFileHandler(logging.FileHandler):
```

Comment:
Is this `customer` and not `custom` intentionaly? I don't know English well enough to be sure